airgun/views/common.py

```diff
@@ -383,6 +386,71 @@ class add_tab(AddTab):
         )
 
 
+class NewAddRemoveResourcesView(View):
+    searchbox = PF4Search()
+    type = Dropdown(
+        locator='.//div[contains(@class, "All repositories") or'
+        ' contains(@aria-haspopup="listbox")]'
+    )
+    Status = Dropdown(
+        locator='.//div[contains(@class, "All") or contains(@aria-haspopup="listbox")]'
+    )
+    add_repo = PF4Button('OUIA-Generated-Button-secondary-2')
```

Comment:
These "generated" locators may not be very stable, is there any alternative, like label?